app/controllers/api/v1/book_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BookController < Api::V1::ApiController
+      def index
+        @books = Book.all
+        render json: @books
+      end
+
+      def show
+        @book = Book.find(params[:id])
```

Comment:
You should use [strong params](https://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html)
Also, what happens if the book_id doesn't exist?